modules/installation-vsphere-infrastructure.adoc

```diff
@@ -45,47 +45,44 @@ endif::[]
 [id="installation-vsphere-infrastructure_{context}"]
 = VMware vSphere infrastructure requirements
 
-You must install the {product-title} cluster on a VMware vSphere version 6 or 7 instance that meets the requirements for the components that you use.
+You must install the {product-title} cluster on a VMware vSphere version 7 instance that meets the requirements for the components that you use.
 
 
 .Version requirements for vSphere virtual environments
 [cols=2, options="header"]
 |===
 |Virtual environment product |Required version
-|VM hardware version | 13 or later
-|vSphere ESXi hosts | 6.5 or later
-|vCenter host   | 6.5 or later
+|VM hardware version | 15 or later
+|vSphere ESXi hosts | 7.0.2 or later
+|vCenter host   | 7.0.2 or later
 |===
 
 [IMPORTANT]
 ====
-Installing a cluster on VMware vSphere version 6.7U3 or earlier and virtual hardware version 13 is now deprecated. These versions are still fully supported, but version 4.11 of {product-title} will require vSphere virtual hardware version 15 or later. Hardware version 15 is now the default for vSphere virtual machines in {product-title}. To update the hardware version for your vSphere nodes, see the "Updating hardware on nodes running in vSphere" article.
+Installing a cluster on VMware vSphere version 7.0.1 or earlier is now deprecated. These versions are still fully supported, but version 4.11 of {product-title} will require vSphere virtual hardware version 15 or later. Hardware version 15 is now the default for vSphere virtual machines in {product-title}. To update the hardware version for your vSphere nodes, see the "Updating hardware on nodes running in vSphere" article.
 
-If your vSphere nodes are below hardware version 15 or your VMware vSphere version is earlier than 6.7U3, upgrading from {product-title} 4.10 to {product-title} 4.11 is not available.
+If your vSphere nodes are below hardware version 15 or your VMware vSphere version is earlier than 7.0.1, upgrading from {product-title} 4.10 to {product-title} 4.11 is not available.
```

Comment:
@duanwei33 ocp 4.10 cluster on vsphere which version is earlier than 7.0.1 is not allowed to upgrade to 4.11?

Reply:
I tested the upgrade with 6.7.3, seems we are **allowed** to upgrade from 4.10 to 4.11. I'm not sure if the doc description is accurate.
Need @gnufied @jsafrane to double confirm.

```
$ oc adm upgrade
Cluster version is 4.10.0-0.nightly-2022-07-28-202108

Upstream is unset, so the cluster will use an appropriate default.
Channel: stable-4.10
warning: Cannot display available updates:
  Reason: VersionNotFound
  Message: Unable to retrieve available updates: currently reconciling cluster version 4.10.0-0.nightly-2022-07-28-202108 not found in the "stable-4.10" channel
```

Reply:
This is not correct, IMO 6.7u3 is still OK in 4.11

Reply:
(at least from storage perspective, I am not sure about OS and networking requirements)

Reply:
yes, here I think we should change to 6.7u3.